devito/types/tensor.py

```diff
@@ -190,33 +192,44 @@ def values(self):
         else:
             return super(TensorFunction, self).values()
 
-    def div(self, shift=None):
+    def div(self, shift=None, order=None):
         """
         Divergence of the TensorFunction (is a VectorFunction).
         """
         comps = []
         func = vec_func(self)
         ndim = len(self.space_dimensions)
         shift_x0 = make_shift_x0(shift, (ndim, ndim))
+        order = order or self.space_order
         for i in range(len(self.space_dimensions)):
             comps.append(sum([getattr(self[j, i], 'd%s' % d.name)
-                              (x0=shift_x0(shift, d, i, j))
+                              (x0=shift_x0(shift, d, i, j), fd_order=order)
                               for j, d in enumerate(self.space_dimensions)]))
         return func._new(comps)
 
     @property
     def laplace(self):
+        """
+        Laplacian of the TensorFunction.
```

Comment:
Maybe add a note to the docstring about `laplace` vs `laplacian`?

Reply:
done